### PR TITLE
Minor updates

### LINKED ISSUE #14: Minor updates
- add some better logging
- add docstrings where applicable
- fix defaults
- fix requirements

### PR DESCRIPTION
**Pull request by nikola-rados**
It may seem like a lot has changed but I've just tried to reorganize some parts of the codebase. The modules and module structure didn't pan out the way that we had originally planned. Namely the `request` and `reconstruct` stages occur at the same time. As such I've changed some things around to better reflect the actual workflow of the code. 

Resolves #14.

### CODE REVIEW

File: orca/requester.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import logging
+from xarray import open_mfdataset, open_dataset
+import re
+from math import ceil
+
+
+logger = logging.getLogger("scripts")
```
Comment (cairosanders) [inline orca/requester.py:7]: :+1:

File: orca/requester.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    logger.debug("File writing complete")
+
+
+def build_url(thredds_base, filepath, variable, lat, lon):
+    """Construct url for OPeNDAP"""
+    return f"{thredds_base}{filepath}?{variable}{lat}{lon}"
+
+
+def split_url(url, size, threshold=5e8):
+    """Based on the request range, split request into edible chunks for THREDDS
```
Comment (cairosanders) [inline orca/requester.py:27]: I like the docstrings

File: Makefile [hunk 3]
```diff
@@ -34,3 +34,9 @@ test: venv
 .PHONY: venv
 venv:
 	test -d $(VENV_PATH) || python3 -m venv $(VENV_PATH)
+
+.PHONY: performance
+performance: venv install
```
Comment (cairosanders) [inline Makefile:39]: Does this just return the time it takes to complete??

**Review by cairosanders — APPROVED** [review]
Looks good to me! I think the reorganization is much cleaner and I always appreciate a good docstring!

File: Makefile (hunk 3 above)
Comment (nikola-rados) [reply Makefile:39]: It will run a general test case and then produce a `snakeviz` runtime page (the same one we were looking at earlier today).

**Pull request merged**